src/css/layout/filterbar.css

```diff
@@ -0,0 +1,163 @@
+}
+
+/* Check box list */
+.check-list {
+    display: block;
+    position: relative;
+    padding-left: 2.9rem;
+    margin-bottom: 6px;
+    cursor: pointer;
+    font-size: 1.5rem;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+  }
+  
+  /* Hide the browser's default checkbox */
+  .check-list input {
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+    height: 0;
+    width: 0;
+  }
+  
+  /* Create a custom checkbox */
+  .checkmark {
+    position: absolute;
+    top: 0;
+    left: 0;
+    border: 1px solid var(--color-primary);
+    height: 1.7rem;
+    width: 1.7rem;
+    background-color: #eee;
+  }
+  
+  /* On mouse-over, add a grey background color */
+  .check-list:hover input ~ .checkmark {
+    background-color: #ccc;
+  }
+  
+  /* When the checkbox is checked, add a blue background */
+  .check-list input:checked ~ .checkmark {
+    background-color: var(--color-primary);
+  }
+  
+  /* Create the checkmark/indicator (hidden when not checked) */
+  .checkmark:after {
+    content: "";
+    position: absolute;
+    display: none;
+  }
+  
+  /* Show the checkmark when checked */
+  .check-list input:checked ~ .checkmark:after {
+    display: block;
+  }
+  
+  /* Style the checkmark/indicator */
+  .check-list .checkmark:after {
+    left: 6px;
+    top: 3px;
+    width: 4px;
+    height: 7px;
+    border: solid white;
+    border-width: 0 1.9px 1.9px 0;
+    -webkit-transform: rotate(45deg);
+    -ms-transform: rotate(45deg);
+    transform: rotate(45deg);
+  }
+
+  /* Radio Buttons */
+  .radio-list {
+    display: block;
+    position: relative;
+    padding-left: 2.9rem;
+    margin-bottom: 6px;
+    cursor: pointer;
+    font-size: 1.5rem;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+  }
+  
+  /* Hide the browser's default radio button */
+  .radio-list input {
+    position: absolute;
+    opacity: 0;
+    cursor: pointer;
+  }
+  
+  /* Create a custom radio button */
+  .radio {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: 1.7rem;
+    width: 1.7rem;
+    border: 1px solid var(--color-primary);
+
+    background-color: #eee;
+    border-radius: 50%;
+  }
+  
+  /* On mouse-over, add a grey background color */
+  .radio-list:hover input ~ .radio {
+    background-color: #ccc;
+  }
+  
+  /* When the radio button is checked, add a blue background */
+  .radio-list input:checked ~ .radio {
+    background-color: var(--color-primary);
+  }
+  
+  /* Create the indicator (the dot/circle - hidden when not checked) */
+  .radio:after {
+    content: "";
+    position: absolute;
+    display: none;
+  }
+  
+  /* Show the indicator (dot/circle) when checked */
+  .radio-list input:checked ~ .radio:after {
+    display: block;
+  }
+  
+  /* Style the indicator (dot/circle) */
+  .radio-list .radio:after {
+       top: 5px;
+      left: 5.3px;
```

Comment:
Any particular reason to use decimal pixel value?

Reply:
It is for the placement of the radio button